fusion-studio-extension/src/browser/core.ts

```diff
@@ -1321,7 +1321,9 @@ export class FSCore {
       node.uri = name;
     }
     node.nodeName = this.getName(name);;
+    delete(this.dict[node.nodeId]);
```

Comment:
please link open issues related to the fix from the commit message, if there isn't an open issue already create a new one